packages/contracts/test/plugins/utils/addresslist.ts

```diff
@@ -122,28 +129,33 @@ describe('AddresslistMock', function () {
       let tx2 = await addresslist.removeAddresses([signers[0].address]);
       await ethers.provider.send('evm_mine', []);
 
-      expect(tx1.blockNumber).to.be.lt(tx2.blockNumber);
+      const rc1 = await tx1.wait();
+      const rc2 = await tx2.wait();
```

Comment:
could be shortened to
```suggestion
      const [rc1, rc2] = await Promise.all([tx1.wait(), tx2.wait()]);
```